web/api/wrapper.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+// WrapperCommand represents components necessary for OAuth2l request
+type WrapperCommand struct {
+	RequestType string
```

Comment:
nit: Consider naming this "CommandType" instead to be consistent with oauth2l main.go